QGCCommon.pri

```diff
@@ -157,7 +157,7 @@ MacBuild {
     CONFIG += x86_64
     CONFIG -= x86
     QMAKE_MACOSX_DEPLOYMENT_TARGET = 10.6
-    QMAKE_MAC_SDK = macosx10.9
+    QMAKE_MAC_SDK = macosx10.11
```

Comment:
Doesn't work. If I remove the line it wants to use 10.10 and Xcode 7.0 only has 10.11.